nRF Toolbox/BGM/StandarGlucoseMonitoring/BGMViewController.swift

```diff
@@ -37,10 +37,12 @@ class BGMViewController: BaseViewController ,CBCentralManagerDelegate, CBPeriphe
     var bgmRecordAccessControlPointCharacteristic       : CBCharacteristic?
     var readings                                        : [GlucoseReading]
     var dateFormatter                                   : DateFormatter
+    
     var bgmServiceUUID                                  : CBUUID
     var bgmGlucoseMeasurementCharacteristicUUID         : CBUUID
     var bgmGlucoseMeasurementContextCharacteristicUUID  : CBUUID
     var bgmRecordAccessControlPointCharacteristicUUID   : CBUUID
```

Comment:
Record Access Control Point may be used in various services. In nRF Toolbox it's used in CGM (Continuous Glucose Measurement) as well, not only bgm.